compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -120,8 +127,16 @@ fun Font(
     identity: String,
     getData: () -> ByteArray,
     weight: FontWeight = FontWeight.Normal,
-    style: FontStyle = FontStyle.Normal
-): Font = LoadedFont(identity, getData, weight, style)
+    style: FontStyle = FontStyle.Normal,
+): Font = LoadedFont(identity, getData, weight, style, FontVariation.Settings())
+
+fun Font(
+    identity: String,
+    getData: () -> ByteArray,
+    weight: FontWeight = FontWeight.Normal,
+    style: FontStyle = FontStyle.Normal,
+    variationSettings: FontVariation.Settings = FontVariation.Settings(weight, style),
+): Font = LoadedFont(identity, getData, weight, style, variationSettings)
```

Comment:
Sorry, I referred to the implementation on Android but didn't think it through carefully. 🙇

Because there are two methods on Android, the difference lies in whether there are variationSettings.
``` kotlin
@Stable
fun Font(
    resId: Int,
    weight: FontWeight = FontWeight.Normal,
    style: FontStyle = FontStyle.Normal,
    loadingStrategy: FontLoadingStrategy = FontLoadingStrategy.Blocking
): Font = ResourceFont(resId, weight, style, FontVariation.Settings(), loadingStrategy)

@ExperimentalTextApi
fun Font(
    resId: Int,
    weight: FontWeight = FontWeight.Normal,
    style: FontStyle = FontStyle.Normal,
    loadingStrategy: FontLoadingStrategy = FontLoadingStrategy.Blocking,
    variationSettings: FontVariation.Settings = FontVariation.Settings(weight, style)
): Font = ResourceFont(resId, weight, style, variationSettings, loadingStrategy)
```
Looking at it now, methods with variationSettings should replace methods without variationSettings in the future.

Users of variable fonts will only choose the second method, while for regular fonts, users can choose either method. This is because if it is not a variable font, the settings for variationSettings will be omitted, which is also fine.

So whether the user is using regular fonts or variable fonts, the methods with variationSettings will work normally.

Therefore, I will only keep the methods with variationSettings. Thank you.

Reply:
As I see, the difference here is when variation settings is empty then the clone function doesn't make a clone of the font